### DISCUSSION TITLE
If stm32duino supports CMSIS-DAP?

### DISCUSSION THREAD

**Discussion by ericzhang-debug** [stm32duino/Arduino_Core_STM32#1915] [Q&A] [answered]
Hello, guys,
CMSIS-DAP is open-source and it is applied more in STM32. 
If stm32duino  support CMSIS-DAP?

Thank you.

**Comment by fpistm** [comment, marked as answer]
Hi,

never try.
Some CMSIS DAP files are available within the CMSIS package provided but I guess it will require some update to the core to be able to use it (include path, wrap source,...) like it was done to support CMSIS DSP.
